core/domain/skill_services.py

```diff
@@ -648,55 +647,6 @@ def update_skill(committer_id, skill_id, change_list, commit_message):
     create_skill_summary(skill.id)
 
 
-def publish_skill(skill_id, committer_id):
```

Comment:
Since, skills are always public. Even now, as soon as a skill is created, in the create function itself, we 'publish' it. This was done as a temporary measure and now, that is permanently changed.
This change was done since the Create Skill modal itself had all the required fields that needed to be filled before a skill could be published, hence no point in having a private skill.